src/controllers/__tests__/item.test.js

```diff
@@ -252,45 +345,57 @@ describe('Items Endpoints', () => {
     });
   });
 
-  describe('GET /api/items/filter', () => {
-    test('Should send only the items with the requested type', async () => {
-      const response = await request(server).get(
-        `/api/items/filter?type=Stationery`
-      );
-
-      const responseBody = response.body;
+  describe('PUT /api/items/:id', () => {
+    test('Should update name on matching item', async () => {
+      const response = await request(server)
+        .put(`/api/items/${itemId}`)
+        .send({ name: 'Couch' });
```

Comment:
@Shrreya could you give your comments on this? I couldn't find something useful in the internet regarding this. But it should not be a problem right, because I am getting content type as json on 'expect' already without setting it?

Reply:
I know, the question here is does it need to be set all the time or not and what is the difference between setting and not..

Reply:
Since you have set `app.use(express.json())` in your Express app configuration, it is able to parse JSON payloads in request bodies even if you haven't explicitly set the content type here when executing the API calls using supertest.

Usually from the client-side it is best to set the Content Type before sending the request because the client doesn't know if the backend is using a JSON parser middleware. But since you're working on the backend, it is alright if you don't add it in some of your tests.

Reply:
@Shrreya Thank you for the explanation🙏🙏

Reply:
Thanks @Shrreya, that explains a lot of things 😅

Reply:
My pleasure! 🙂